contributed_definitions/nyaml/NXenergydispersion.yaml

```diff
@@ -57,6 +57,22 @@ NXenergydispersion(NXobject):
     doc: |
       Individual lenses in the energy dispersive section
   (NXfabrication):
+  (NXtransformations):
+    depends_on(NX_CHAR):
```

Comment:
The `depends_on` field should be directly in the NXenergydispersion without an addtional NXtransformation

Reply:
Was this removed? I don't see it anymore.

Reply:
Looks like it. It says also Outdated here

Reply:
Ok, then I'll just add it again directly in the NXenergydispersion

Reply:
But I just merged NXcalibrations from fairmat 🤔

Reply:
True, I don't know where it was changed. Anyway, I have added it again.